guide/payments/send.md

```diff
@@ -39,169 +109,184 @@ layout = "full-width"
 
 # Sending bitcoin
 
-Sending bitcoin can be a very straightforward or complex flow in a Bitcoin application. People may be sending bitcoin to a known contact, moving it between their wallets on different devices, or making a purchase through a [payment processor]({{ '/guide/getting-started/software/#payment-processors' | relative_url }}).
+Sending bitcoin can be a very simple, or complex flow in a Bitcoin application. People may want to send bitcoin to a known contact, transfer it to another wallet on a different device, or make a purchase via a [payment processor]({{ '/guide/getting-started/software/#payment-processors' | relative_url }}).
 
-Let us look at the main transaction options senders need to configure when moving bitcoin:
+## Payment entry points
 
-- **Amount** — How much to send
-- **Recipient address** — Where to send the bitcoin
-- **Coin selection** — Which coins/inputs to use (optional)
-- **Fee settings** — Prioritize fast confirmation or low cost (optional)
+The need to send bitcoin can be triggered by many different use cases, and initiated by both sender and receiver. Common entry points are:
 
-You do not need to follow the order below. Feel free to tailor the configuration's order for the payment to what best serves your users. For example, you may make users set the amount before they enter the address.
+- Manual entry of address and amount
```

Comment:
What's the difference between:
- Manual entry of address and amount and 
- Copy & pasting payment information?

Reply:
Manual entry is the most basic entry form and has to be supported.

Copy & paste can be an interaction with an input field, but ideally, the app auto-detects clipboard content and presents the appropriate UI automatically.

Reply:
Does it have to be supported though? I can't imagine entering it like that and if they did it has major room for error.